doc/news/changes/incompatibilities/20171020DavidWells

```diff
@@ -0,0 +1,3 @@
+Changed: The <code>enum</code> VectorOperation is now declared in its own header instead of the header for Vector.
```

Comment:
I think we only use ``incompatibilities/`` if it is something that users updating from an older release need to be aware of, which is not the case here.